cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -494,6 +494,103 @@ func Test_certManagerClient_deleteObjs(t *testing.T) {
 	}
 }
 
+func Test_certManagerClient_PlanUpgrade(t *testing.T) {
+
+	tests := []struct {
+		name         string
+		objs         []runtime.Object
+		expectErr    bool
+		expectedPlan CertManagerUpgradePlan
+	}{
+		{
+			name: "returns the upgrade plan for cert-manager",
```

Comment:
might want to add a test for update form vX (explicitly set) to the embedded version